build/ci-Dockerfile

```diff
@@ -1,11 +1,11 @@
 # Build the manager binary
-FROM quay.io/konveyor/builder:v1.19 AS builder
+FROM quay.io/konveyor/builder AS builder
```

Comment:
wouldn't be good to use a fixed version? (a future update from the image may cause the image to brake)

Reply:
@mateusoliveira43 We've been moving towards latest since this is the upstream builder. Most of our other repos already do this. That way we automatically get fixes, and if things break due to this, we can fix them earlier.